ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -161,6 +163,7 @@ class NewTabPageViewController: UIViewController {
     )
     self.p3aHelper = p3aHelper
     background = NewTabPageBackground(dataSource: dataSource)
+    videoBackgroundController = NewTabPageVideoBackgroundController(background: background)
```

Comment:
A new video player is created for every new tab. Is this expected?

Reply:
no, I will change the code to create player only for background with video NTT